packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3469,14 +3467,13 @@ export class ContainerRuntime
 					versionId: versionId != null ? versionId : undefined,
 					closeSummarizerDelayMs: this.closeSummarizerDelayMs,
 				},
-				error,
+				new GenericError("Restarting summarizer instead of refreshing"),
```

Comment:
The `summarize_cancel` will still be sent twice right? Because essentially this second attempt fails. Or does this change fix that too?

Reply:
The second summarize cancel becomes generic because the container is disconnected